source/technical/images.rst

```diff
@@ -25,9 +25,26 @@ To make a snapshot of a bare-metal instance, run the following command from insi
 .. code-block:: bash
 
    sudo cc-snapshot <image_name>
+   
+.. tip:: 
+   You may get warnings, such as "image too large", during the snapshotting, and get prompted to confirm. If you are confident about what you are trying to do, you can skip all warnings by using ``-f`` flag. 
+   
+   .. code-block:: bash
+   
+      sudo cc-snapshot -f <image_name>
+      
+   In addition, you can exclude directories by using the ``-e`` flag.
+   
+   .. code-block:: bash
+   
+      sudo cc-snapshot -e <dir1>,<dir2> <image_name>
```

Comment:
This is not how the -e option should be used: I just tried it locally and I believe it should be `-e dir1 -e dir2`. Can you please try it on an actual Chameleon instance?